src/components/molecules/NotificationMarkdown/NotificationMarkdown.tsx

```diff
@@ -0,0 +1,19 @@
+import ReactMarkdown from 'react-markdown';
+
+import {openUrlInExternalBrowser} from '@utils/shell';
+
+const NotificationMarkdown = ({message}: any) => {
+  return (
+    <ReactMarkdown
+      components={{
+        a(prps: any) {
```

Comment:
Do they export any type that we could use for the props here?
Also small typo "prps"